src/__tests__/store.js

```diff
@@ -271,7 +264,9 @@ describe('Store', () => {
     };
 
     // Make sure distinct parameter is gone when overridden with undefined.
-    expect(store.queryParameters).not.toHaveProperty('distinct');
+    expect(store.queryParameters).toEqual(
+      expect.objectContaining({ distinct: undefined })
```

Comment:
Re-using the helper logic on this one, and kept support for `null`.
`null` gets transformed into `undefined` by the store, and `undefined` will make the helper remove it from the call.